nilearn/decomposition/tests/test_canica.py

```diff
@@ -67,5 +90,34 @@ def test_canica_square_img():
     # Smoke test to make sure an error is raised when no data is passed.
     assert_raises(TypeError, canica.fit)
 
+
+def test_component_sign():
+    # BF for issue #570
```

Comment:
> > +def test_component_sign():
> > +    # BF for issue #570

> Mention "regression test" at this point? ...perhaps instead of "BF".

Good point. And don't put a number, which doesn't mean much to the
reader, but a short description of the issue.